### PR TITLE
gh-85583: Add f-string index in tutorial/inputoutput.rst 

### PR DESCRIPTION
**Pull request by amaajemyfren**
https://bugs.python.org/issue41411

* Issue: gh-85583

### CODE REVIEW

**Comment by ILMostro** [comment]
Relates to #21552, which seems to be inexplicably at a stalemate.

**Review by AA-Turner — APPROVED** [review]

**Comment by AA-Turner** [comment]
On second thought, this might ought go in lexical analysis.

A

File: Doc/tutorial/inputoutput.rst [hunk 1]
```diff
@@ -100,7 +100,13 @@ yet another way to substitute values into strings, using placeholders like
...
 control of the formatting.
 
-
+.. index::
+   single: formatted string literal
+   single: interpolated string literal
+   single: string; formatted literal
+   single: string; interpolated literal
+   single: f-string
+   single: fstring
```
Comment (willingc) [inline Doc/tutorial/inputoutput.rst:109]:
```suggestion
   single: fstring

```

**Review by willingc — APPROVED** [review]

**Comment by willingc** [comment]
I'm going to merge as is. If @AA-Turner would like, please create a new PR to move to lexical analysis.

Thanks @amaajemyfren for the contribution. :sunny:

**Pull request merged**